http-core/src/main/scala/org/apache/pekko/http/scaladsl/model/StatusCode.scala

```diff
@@ -151,8 +151,10 @@ object StatusCodes extends ObjectRegistry[Int, StatusCode] {
   val Gone                         = reg(c(410)("Gone", "The resource requested is no longer available and will not be available again."))
   val LengthRequired               = reg(c(411)("Length Required", "The request did not specify the length of its content, which is required by the requested resource."))
   val PreconditionFailed           = reg(c(412)("Precondition Failed", "The server does not meet one of the preconditions that the requester put on the request."))
+  val ContentTooLarge              = reg(c(413)("Content Too Large", "The request content is larger than the server is willing or able to process."))
+  @deprecated("deprecated in favor of ContentTooLarge", "1.0.0")
   val PayloadTooLarge              = reg(c(413)("Payload Too Large", "The request payload is larger than the server is willing or able to process."))
-  @deprecated("deprecated in favor of PayloadTooLarge", "Akka HTTP 10.1.11")
+  @deprecated("deprecated in favor of ContentTooLarge", "1.0.0")
   val RequestEntityTooLarge        = PayloadTooLarge
```

Comment:
can you change this to this
```
@deprecated("deprecated in favor of PayloadTooLarge, which was later deprecated in favor of ContentTooLarge (please switch to ContentTooLarge)", "Akka HTTP 10.1.11")
```


* can you a comment - `// RequestEntityTooLarge was originally deprecated in favour of PayloadTooLarge in Akka HTTP 10.1.11` ?
* This will remind us that this line can actually be removed soon - possibly in Pekko 1.1.0 - because this line was already deprecated - but it does make sense to adjust the deprecation message now